core/src/main/java/hudson/triggers/Trigger.java

```diff
@@ -334,4 +345,7 @@ public void run(AbstractProject p) {
         }
         return r;
     }
+
+    @Extension
+    public static final SlowTriggerAdminMonitor ADMIN_MONITOR = new SlowTriggerAdminMonitor();
```

Comment:
Why don't you just put extension on the `SlowTriggerAdminMonitor` class? In such configuration the instance will be shared between unit tests, and I doubt it is a desirable behavior